src/components/Accordion/Accordion.tsx

```diff
@@ -64,11 +78,14 @@ const Accordion = ({
   return (
     <div {...classes('', appearance)} {...rest}>
       {addButton ? (
-        <AccordionButtonLine addButton={addButton} appearance={appearance}>
+        <AccordionButtonLine
+          addButton={addButton}
+          appearance={appearance}
+          handleToggle={handleToggle}>
           <Button css={buttonStyle} stripped onClick={handleToggle}>
             {title}
           </Button>
-          {toggleSwitch}
+          <>{toggleSwitch}</>
```

Comment:
Kan fragment fjernes?

Reply:
Dette var en work-around om toggleSwitch var udefinert. Se https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051

Reply:
Åja, det problemet var nytt for meg. Lur løsning 👍